quill-core/src/main/scala/io/getquill/dsl/QueryDsl.scala

```diff
@@ -13,6 +13,9 @@ private[getquill] trait QueryDsl {
   @compileTimeOnly(NonQuotedException.message)
   def querySchema[T](entity: String, columns: (T => (Any, String))*): EntityQuery[T] = NonQuotedException()
 
+  @compileTimeOnly(NonQuotedException.message)
+  def impliedQuerySchema[T](entity: String, columns: (T => (Any, String))*): EntityQuery[T] = NonQuotedException()
```

Comment:
TODO. Needs to be private. We can't use `querySchema` for places where a schema needs to be implicitly inferred (e.g. insert/update/delete) because when `querySchema` goes into the parser, the parser thinks it's an entity with `Entity.renameable = Fixed` which is wrong (when insert/update/delete are used).
TODO Make sure to check in `ActionMacroSpec` there is a test where `querySchema` or `schemaMeta` will actually cause a insert/update/delete query to have a Entity with `renameable = Fixed`.